tests/config/config.d/zookeeper_fault_injection.xml

```diff
@@ -12,7 +12,11 @@
           In other words, session will expire 4 times per 99996 successful requests
           or approximately each 25000 requests (on average).
         -->
+        <enable_fault_injections_during_startup>0</enable_fault_injections_during_startup>
         <send_fault_probability>0.00002</send_fault_probability>
         <recv_fault_probability>0.00002</recv_fault_probability>
+
+        <send_sleep_probability>0.00001</send_sleep_probability>
+        <send_sleep_ms>10000</send_sleep_ms>
```

Comment:
Just a note: it makes sense to try `9900` or `9990`, I guess it will find more interesting bugs